### DISCUSSION TITLE
Report Portal with Python 3.9.1

### DISCUSSION THREAD

**Discussion by den-cezar** [ElSnoMan/pyleniumio#170] [Q&A]
Hi I'm having some issue with RP and pyleniumio.
 
What we have:
- simple test case:
```
def test_failed(py):
    py.visit("https://google.com")
    assert 2 == 3, 'Test failed'
```
- pyleniumio run it with key `--reportportal`

What we get:
Exaction: `ValueError: read of closed file`
![image](https://user-images.githubusercontent.com/17469276/109152311-fcbb2680-7773-11eb-89a4-f6238d3f4d79.png)

With this error launch in RP continue showing as running and screenshot is not loading into it.

RP and pyleniumio used the lasted version and python 3.9.1. Any idea how to fix it ?

**Comment by ElSnoMan** [comment]
This looks like an issue with [Report Portal](https://github.com/reportportal/reportportal) when trying to upload results to it. I'd recommend posting the issue there, but more details can help shed some light on this:

1. Does this error only happen with a test that is using the Pylenium `py` fixture?
2. Does this error only happen with a `failing` test?
3. Does this error only happen with `python 3.9.1`?

**Comment by den-cezar** [comment]
Thanks, I'm try to reproduce in different environments and let you know. 
I'm found it while execute `failing` test because it's loading an image file into Report Portal.
